### PR TITLE
Update author bio

### PR DESCRIPTION
**Pull request by goschevski**
I've updated my bio.

Thanks.

### CODE REVIEW

File: data/authors.yml [hunk 1]
```diff
@@ -200,14 +200,10 @@
...
   website: http://www.goschevski.com/
-  twitter: goschevski
   github: goschevski
   bio: >
-    <p>Aleksandar Goševski is front-end developer based in Belgrade, Serbia. He works at
-      <a href="http://vast.com">Vast.com</a> as JavaScript developer, but he is secretly in love in Sass.</p>
-    <p>You can find him on <a href="http://github.com/goschevski">GitHub</a>,
-      <a href="http://dribbble.com/goschevski">Dribbble</a>, or <a href="http://twitter.com/goschevski">Twitter</a>
-      as @goschevski.</p>
+    <p>Aleksandar Goševski is front-end developer based in Belgrade, Serbia. He works as JavaScript developer, but he is secretly in love in CSS.</p>
```
Comment (jlong) [inline data/authors.yml]: with CSS? or in CSS?

Comment (goschevski) [inline data/authors.yml]: Not sure what's correct. :D

Comment (jlong) [inline data/authors.yml]: In the US, we would say "in love *with* CSS" :)

**Comment by goschevski** [comment]
I fixed it. :) Thanks!

**Comment by jlong** [comment]
Done!

**Comment by goschevski** [comment]
Thanks again!

**Pull request merged**